app/javascript/utilities/connect/getUnopenedChannels.jsx

```diff
@@ -23,9 +23,9 @@ class UnopenedChannelNotice extends Component {
 
   componentDidMount() {
     const { pusherKey } = this.props;
-    const appName = document.body.dataset.appName;
+    const { appDomain } = document.body.dataset;
```

Comment:
oh man, I should have changed this in the original PR -- sorry about that 😞  and thank you for fixing this!

Reply:
that alright!, it was doing no harm.